### ISSUE TITLE
Staging Review finding: Portal sign in link styled to look like a button

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#104050]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Accredited Rep Facing (ARF) + Accredited Rep Mgmt (ARM)
**Product:** ARF (Accredited Rep Portal MVP) + ARM (Appoint a Rep 2.0)
**Feature:** Form 21-22: Digital Submission (ARM) and Accept/Decline (ARF)

## Findings details
**VA.gov Experience Standard - issue:** [User can identify an element and its state.](https://design.va.gov/about/experience-standards/#user-can-identify-an-element-and-its-state)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

There are a couple of instances of elements coded as links but visually styled as buttons:
- On [/representative](https://staging.va.gov/representative) when unauthenticated, the sign in link in the header is styled to look like a button.
- On [/representative/poa-requests](https://staging.va.gov/representative/poa-requests?status=pending&sort=created_at_asc), the pending/processed tabs are links styled to look like buttons.

In general, we try to maintain material honesty on VA.gov --- if it looks like a button, it's coded as a button and it performs an action; if it looks like a link, it's coded as a link and it navigates.

This isn't just an academic exercise. For voice command / speech recognition software users, one of the common interaction patterns is to speak a command like "Click button" and select from all of the buttons that are highlighted on the page. If you expect something to be a button from its visual appearance but it doesn't get highlighted when you expect buttons to be highlighted, you're left to try to guess and troubleshoot what's happening. That's extra friction that's easy to remove by just making links look like links and buttons look like buttons.

<details><summary>Link, screenshot or steps to recreate</summary>

![Header for the accredited rep portal, with sign in button in the top right](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/99796-Brian-3.png?raw=true)

</details>

## Recommended action

The sign in link should be visually styled as a link; or, alternatively, a sign in button can open a modal (as happens on the "regular" VA.gov).

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** N/A
- **Modality:** Voice
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
